completions/spicetify.lua

```diff
@@ -0,0 +1,160 @@
+--------------------------------------------------------------------------------
+-- Clink argmatcher for Spicetify
+--------------------------------------------------------------------------------
+
+function split(inputstr, sep)
+    if sep == nil then
+        sep = "%s"
+    end
+    local t={}
+    for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
+        table.insert(t, str)
+    end
+    return t
+end
+
+function get_extension_names()
+    local handle = io.popen("spicetify.exe path -e")
```

Comment:
You'll want `"2>nul spicetify.exe path -e"` so that any errors are not printed to the screen (which would garble the input line display).

The same suggestion applies to all instances of `io.popen` in this file.

Reply:
Ah okay cool, changed it.